templates/base.html

```diff
@@ -4,6 +4,7 @@
   <title>{{ Configs.ctf_name }}</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="shortcut icon" href="{{ url_for('views.themes', theme='core', path='img/favicon.ico') }}" type="image/x-icon">
```

Comment:
i think the endpoint should be `views.themes_beta` for the core-beta theme for now. Also we should probably be just using the core-beta assets so no need to specify the theme. If you need to you can copy over the favicon file into the assets folder and then modify vite.config.js to copy that into the static folder.

Reply:
Alright. I modified it now. 

I have a question. I am running this theme inside a local docker container and when you open the site in the browser and check where it got the favicon.ico, it says that it got it from core-beta/static/img folder
![image](https://user-images.githubusercontent.com/78736225/180988677-f32b9584-bb86-4c7c-9a2d-85c131ebde3e.png)
But, when I check the files in VSCode it doesn't have the img folder inside the static folder.
![image](https://user-images.githubusercontent.com/78736225/180988922-9d4718f6-a51d-4e04-8f3d-5099f6389d17.png)